core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,32 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.generateDummyExplorations = function() {
+          // Generate dummy explorations with random title.
+          AdminTaskManagerService.startTask();
+          var DummyExpCount = $scope.DummyExplorationCount,
+            DummyExpPublish = $scope.DummyExplorationPublishCount;
+          if (DummyExpPublish > DummyExpCount) {
+            $scope.setStatusMessage(
+            'Publish exploration count should be less than generate count');
+            AdminTaskManagerService.finishTask();
```

Comment:
> Speaking of verifying correctness, can you think of any other error cases that may occur when the user inputs arbitrary strings? Check for those too.

I have added the input type as `number` at front end. So, a user can input numbers only. 

Done.